test/distributed/elastic/launchers/api_test.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python3
```

Comment:
do we plan to plug `api_test.py` to `test/run_test.py` in the same as other distributed tests, or do we plan to use a separate test launching solution. If it's the former and if the tests are supposed to pass today, shall we enable them in this PR. If it's the later, we might need to make sure that we can still create an issue with title format "DISABLED testname.TestClass" to disable it in CI.